### PR TITLE
Custom properties real code snippets examples + rearrange the text

### PR DESCRIPTION
**Pull request by iamwhaler**
http://void-docs.talkable.com/advanced_features/passing_custom_data.html

## Demo

http://void-docs.talkable.com/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-1234)

### CODE REVIEW

File: source/advanced_features/passing_custom_data.rst [hunk 1]
```diff
@@ -4,55 +4,75 @@
...
+across all Campaign Views.
 
-The values you define for some key should be JSON Key-Value pairs themselves,
-this means complex data structures can be passed through.
 
-Any ``custom_properties`` data passed through is tied to the |advocate| or the |friend|,
-that means there is only one instance of each field in
-existence at any time: the most recently passed data.
+How to set
+==============
```
Comment (zhuravel) [inline source/advanced_features/passing_custom_data.rst]:
```suggestion
==========
```

File: source/advanced_features/passing_custom_data.rst [hunk 2]
```diff
@@ -4,55 +4,75 @@
...
-Key Value pairs can be referenced through your newly created object by referencing
-the value name such as:
+
+.. note::
+
+    The values you pass in the ``custom_properties`` have to be JSON Key-Value pairs themselves,
+    meaning that complex nested data structures can not be passed through.
+
+How to use
+==============
```
Comment (zhuravel) [inline source/advanced_features/passing_custom_data.rst]:
```suggestion
==========
```

File: source/advanced_features/passing_custom_data.rst [hunk 3]
```diff
@@ -4,55 +4,75 @@
...
+To access ``custom_properties`` in Talkable, use:
 
 .. code-block:: liquid
 
-   {{ data.favorite_color }}
+   {{ advocate_custom_properties }}
+   {{ friend_custom_properties }}
 
-Defining ``custom_properties`` in your integration might look something like this:
+Key Value pairs can be referenced calling the desired data key, such as:
```
Comment (zhuravel) [inline source/advanced_features/passing_custom_data.rst]: Let’s be consistent.
```suggestion
Key-Value pairs can be referenced calling the desired data key, such as:
```

**Comment by iurevych** [comment]
@iamwhaler could you provide a Jira ref?

**Comment by iamwhaler** [comment]
@iurevych it wasn't requested by anyone specifically, I attached task for the reference

**Pull request merged**